plugins/inputs/webhooks/webhooks.go

```diff
@@ -0,0 +1,81 @@
+package webhooks
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	_ "github.com/influxdata/telegraf/plugins/inputs/webhooks/webhooks_all"
+	"github.com/influxdata/telegraf/plugins/inputs/webhooks/webhooks_models"
+)
+
+func init() {
+	inputs.Add("webhooks", func() telegraf.Input { return NewWebhooks() })
+}
+
+type Webhooks struct {
+	ServiceAddress string
+
+	Webhook []WebhookConfig
+}
+
+type WebhookConfig struct {
```

Comment:
Oh nevermind, we just find that is how Go works. We just have to use a pointer instead it seems to work then.